sympy/stats/drv_types.py

```diff
@@ -540,6 +540,13 @@ def _characteristic_function(self, t):
     def _moment_generating_function(self, t):
         return exp(self.lamda * (exp(t) - 1))
 
+    def expectation(self, expr, var, evaluate=True, **kwargs):
+        if evaluate:
+            if expr == var:
+                return self.lamda
+            if isinstance(expr, FallingFactorial) and expr.args[1].is_integer and expr.args[1].is_positive and expr.args[0] == var:
```

Comment:
Please break the lines appropriately, as the current lines are too long.  
Also, please add tests related to this code.

Reply:
I have added a test for the E(X) = lambda case. I will add for the rest.